Assorted typo fixes.

XhmikosR opened the pull request:
Signed-off-by: XhmikosR <xhmikosr@gmail.com>

**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [ ] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---
**What does this PR aim to accomplish?:**
*A detailed description, screenshots (if necessary), as well as links to any relevant GitHub issues*


**How does this PR accomplish the above?:**
*A detailed description (such as a changelog) and screenshots (if necessary) of the implemented fix*


**What documentation changes (if any) are needed to support this PR?:**
*A detailed list of any necessary changes*


---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

dschaper commented:
We need an automated spell checker.

XhmikosR commented:
This was automated mostly.

dschaper commented:
Yeah, more of a linter idea. Keep the spells from getting in in the first place.

XhmikosR commented:
I use [misspell](https://github.com/client9/misspell) but it doesn't seem it's maintained a lot.

dschaper commented:
And Promo uses British English with the added 'u's everywhere. I use the American English without the `u` and `z`s.

XhmikosR commented:
```
C:\Users\xmr\Desktop\pi-hole>misspell -locale US -w .
.github\ISSUE_TEMPLATE.md:12:11: corrected "behaviour" to "behavior"
.github\ISSUE_TEMPLATE.md:16:9: corrected "behaviour" to "behavior"
advanced\Scripts\COL_TABLE:1:46: corrected "colours" to "colors"
advanced\Scripts\COL_TABLE:4:25: corrected "emphasised" to "emphasized"
advanced\01-pihole.conf:26:0: corrected "localise" to "localize"
advanced\Scripts\chronometer.sh:75:28: corrected "colours" to "colors"
advanced\Scripts\chronometer.sh:91:32: corrected "Visualise" to "Visualize"
advanced\Scripts\chronometer.sh:134:10: corrected "Colour" to "Color"
advanced\Scripts\chronometer.sh:272:33: corrected "behaviour" to "behavior"
advanced\Scripts\chronometer.sh:311:16: corrected "colour" to "color"
LICENSE:15:22: corrected "LICENCE" to "LICENSE"
LICENSE:19:27: corrected "Licence" to "License"
LICENSE:19:127: corrected "Licence" to "License"
LICENSE:19:171: corrected "authorised" to "authorized"
LICENSE:19:193: corrected "Licence" to "License"
LICENSE:20:45: corrected "Licence" to "License"
LICENSE:26:8: corrected "Licence" to "License"
LICENSE:28:6: corrected "Licence" to "License"
advanced\Scripts\piholeCheckout.sh:39:57: corrected "cancelled" to "canceled"
advanced\Scripts\setupLCD.sh:23:14: corrected "Unrecognised" to "Unrecognized"
LICENSE:28:20: corrected "Licence" to "License"
LICENSE:29:97: corrected "Licence" to "License"
LICENSE:30:141: corrected "Licence" to "License"
LICENSE:34:96: corrected "Licence" to "License"
LICENSE:35:78: corrected "Licence" to "License"
LICENSE:36:106: corrected "Licence" to "License"
LICENSE:39:38: corrected "Licence" to "License"
LICENSE:41:87: corrected "licence" to "license"
LICENSE:50:161: corrected "licence" to "license"
LICENSE:51:195: corrected "Licence" to "License"
LICENSE:59:16: corrected "Licence" to "License"
LICENSE:65:128: corrected "Licence" to "License"
LICENSE:65:240: corrected "Licence" to "License"
LICENSE:67:183: corrected "Licence" to "License"
LICENSE:67:221: corrected "Licence" to "License"
LICENSE:67:310: corrected "Licence" to "License"
LICENSE:67:529: corrected "Licence" to "License"
advanced\blockingpage.css:328:16: corrected "colour" to "color"
LICENSE:69:171: corrected "Licence" to "License"
LICENSE:69:262: corrected "Licence" to "License"
LICENSE:69:311: corrected "Licence" to "License"
LICENSE:69:333: corrected "licences" to "licenses"
LICENSE:69:382: corrected "Licence" to "License"
LICENSE:69:446: corrected "Licence" to "License"
LICENSE:69:499: corrected "Licence" to "License"
LICENSE:69:542: corrected "Licence" to "License"
LICENSE:73:23: corrected "Licence" to "License"
LICENSE:77:189: corrected "Licence" to "License"
LICENSE:79:193: corrected "Licence" to "License"
LICENSE:81:25: corrected "Licence" to "License"
LICENSE:81:96: corrected "licence" to "license"
LICENSE:81:164: corrected "Licence" to "License"
advanced\dnsmasq.conf.original:49:38: corrected "favour" to "favor"
LICENSE:86:53: corrected "Licence" to "License"
LICENSE:86:356: corrected "Licence" to "License"
LICENSE:87:56: corrected "Licence" to "License"
LICENSE:91:215: corrected "Licence" to "License"
LICENSE:95:135: corrected "Licence" to "License"
advanced\dnsmasq.conf.original:407:27: corrected "recognise" to "recognize"
advanced\index.php:9:3: corrected "Sanitise" to "Sanitize"
LICENSE:97:22: corrected "Licence" to "License"
LICENSE:99:23: corrected "Licence" to "License"
advanced\dnsmasq.conf.original:414:48: corrected "recognised" to "recognized"
advanced\index.php:71:67: corrected "authorised" to "authorized"
LICENSE:99:142: corrected "Licence" to "License"
LICENSE:99:326: corrected "Licence" to "License"
LICENSE:100:39: corrected "Licence" to "License"
LICENSE:100:144: corrected "Licence" to "License"
LICENSE:104:343: corrected "Licence" to "License"
advanced\index.php:218:69: corrected "localised" to "localized"
LICENSE:106:23: corrected "Licence" to "License"
automated install\uninstall.sh:17:71: corrected "cancelled" to "canceled"
LICENSE:108:4: corrected "Licence" to "License"
LICENSE:108:126: corrected "Licence" to "License"
LICENSE:109:42: corrected "licences" to "licenses"
LICENSE:109:119: corrected "Licence" to "License"
LICENSE:109:185: corrected "Licence" to "License"
automated install\uninstall.sh:18:67: corrected "cancelled" to "canceled"
LICENSE:113:42: corrected "Licence" to "License"
LICENSE:114:24: corrected "Licence" to "License"
LICENSE:114:141: corrected "Licence" to "License"
LICENSE:115:86: corrected "Licence" to "License"
LICENSE:115:227: corrected "Licence" to "License"
LICENSE:115:256: corrected "Licence" to "License"
LICENSE:116:32: corrected "Licence" to "License"
LICENSE:127:7: corrected "Licence" to "License"
LICENSE:128:7: corrected "licence" to "license"
LICENSE:134:11: corrected "Licences" to "Licenses"
LICENSE:140:17: corrected "Licence" to "License"
LICENSE:141:28: corrected "Licence" to "License"
LICENSE:143:24: corrected "Licence" to "License"
LICENSE:144:31: corrected "Licence" to "License"
LICENSE:145:82: corrected "licences" to "licenses"
LICENSE:145:200: corrected "Licence" to "License"
pihole:305:4: corrected "Colour" to "Color"
pihole:306:4: corrected "Colour" to "Color"
pihole:307:4: corrected "Colour" to "Color"
automated install\basic-install.sh:822:10: corrected "Cancelling" to "Canceling"
automated install\basic-install.sh:828:10: corrected "Cancelling" to "Canceling"
```

XhmikosR commented:
I kept it as a separate patch in case you want to revert it.

XhmikosR commented:
Oh, and I didn't touch LICENSE.

PromoFaux commented on gravity.sh:
```diff
@@ -464,7 +464,7 @@ gravity_ParseFileIntoDomains() {
   #if [[ "${source}" == "${piholeDir}/${matterAndLight}" ]]; then
     # Remove comments and print only the domain name
-    # Most of the lists downloaded are already in hosts file format but the spacing/formating is not contigious
+    # Most of the lists downloaded are already in hosts file format but the spacing/formating is not contagious
```
```suggestion
    # Most of the lists downloaded are already in hosts file format but the spacing/formating is not 
contiguous
```

![image](https://user-images.githubusercontent.com/1998970/74687981-ee841300-51cd-11ea-82b6-9beaee02f30c.png)

dschaper commented:
Stickler lint is unrelated.

pralor commented:
This pull request has been mentioned on **Pi-hole Userspace**. There might be relevant details there:

https://discourse.pi-hole.net/t/pi-hole-5-1-released/35577/1
